testnets/testnets.md

```diff
@@ -17,15 +17,15 @@ Let's start rolling your node and start producing blocks in live testnet environ
 
 First of all make sure you followed the installation steps in [build requirements section](./build-requirements.md). You should have the required binaries. If you just want to copy and execute the scripts below, make sure to set up environment variables:
 
-Below is the [demonet configuration](https://github.com/CosmWasm/testnets/blob/master/demo-09/config).
+Below is the [demonet configuration](https://github.com/CosmWasm/testnets/blob/master/demo-10/config).
```

Comment:
Does this file exist? I don't see that directory added to the repo?

Reply:
https://github.com/CosmWasm/testnets/pull/5